ddtrace/opentracer/tracer.py

```diff
@@ -1,13 +1,60 @@
 import opentracing
+import logging
+
+from ddtrace import Tracer as DatadogTracer
+from ddtrace.constants import FILTERS_KEY
+from ddtrace.ext import AppTypes
+from ddtrace.sampler import AllSampler
+from ddtrace.settings import ConfigException
+
+from .constants import *
+from .util import merge_dicts
+
+log = logging.getLogger(__name__)
+
+DEFAULT_CONFIG = {
+    AGENT_HOSTNAME_KEY: 'localhost',
+    AGENT_PORT_KEY: 8126,
+    DEBUG_KEY: False,
+    ENABLED_KEY: True,
+    GLOBAL_TAGS_KEY: {},
+    SERVICE_NAME_KEY: None,
+    SAMPLER_KEY: AllSampler(),
```

Comment:
it might be nice to keep the config to a json (or whatever other serialization) format